### PR TITLE
Fix/issues from testing

### PR DESCRIPTION
**Pull request by shambupujar**
# Pull Request Template

Add ansible playbook for optimising node that setups up limits and swap space
Fixes for the issues observed during the testing
Refactored code for systemd and docker into separate files

### CODE REVIEW

**Review by RayOffiah — DISMISSED** [review]
Looks fine to me.

File: monitoring/node-monitoring.yml [hunk 1]
```diff
@@ -27,7 +27,7 @@ services:
     environment:
       METRICS_EXPORTER_BASIC_AUTH: "${BASIC_AUTH_USER_CREDENTIALS}"
       METRICS_EXPORTER_ROOT_API_URL: "${NODE_END_POINT}"
-      METRICS_EXPORTER_LEDGER_FOLDER: /data
+      #METRICS_EXPORTER_LEDGER_FOLDER: /data
```
Comment (rvelaz) [inline monitoring/node-monitoring.yml:30]: is this commented out on purpose?

Comment (shambupujar) [reply monitoring/node-monitoring.yml:30]: Yeah.. as that was throwing exception when user doesn't setup /data directory. Also the dashboard which we sharing is not picking up any info.. I will remove the line itself

**Review by RayOffiah — APPROVED** [review]
New ansible check looks okay from here.

**Review by RayOffiah — APPROVED** [review]

**Pull request merged**